### DISCUSSION TITLE
Check hwcodec config, exit with code 0

### DISCUSSION THREAD

**Discussion by zdavatz** [rustdesk/rustdesk#6954] [General]
### Bug Description

Start Rusdeck 1.2.3 and then try to connect. ID is not found

### How to Reproduce

Try to connect.

### Expected Behavior

I should be able to connect.

### Operating system(s) on local side and remote side

Windows 10

### RustDesk Version(s) on local side and remote side

1.2.3

### Screenshots

![log_dr](https://github.com/rustdesk/rustdesk/assets/4953/66154c4d-4900-43b7-8f0b-b34b8b6590ce)

**Comment by rustdesk** [comment]
self hosting please.

**Comment by zdavatz** [comment]
Selfhosting did not help. Same problem still exists. Client can not connect. Can I turn on some debugging in the client?
